src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Synthesized.cs

```diff
@@ -1202,7 +1193,7 @@ public void AppliedCompilationRelaxationsOnModuleSupressesAssemblyAttributes(Out
 using System.Runtime.CompilerServices;
 
 [assembly: CompilationRelaxationsAttribute(0)]
-", options: new CSharpCompilationOptions(OutputKind.NetModule, optimizationLevel: optimizationLevel));
+", options: TestOptions.CreateTestOptions(outputKind, optimizationLevel));
```

Comment:
While reviewing the changes again, I think my change here was unintentional. The original test was using NetModule and I used the given `outputKind`.

```suggestion
", options: TestOptions.CreateTestOptions(OutputKind.NetModule, optimizationLevel));
```